schemas/2018-04-01/Microsoft.Network.json

```diff
@@ -0,0 +1,547 @@
+{
+    "id": "https://schema.management.azure.com/schemas/2018-04-01/Microsoft.Network.json#",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Microsoft.Network",
+    "description": "Microsoft Network Resource Types",
+    "resourceDefinitions": {
+      "trafficmanagerprofiles": {
```

Comment:
>trafficmanagerprofiles [](start = 7, length = 22)

Please use consistent resource type names.. In the root template it's "trafficManagerProfiles"